### PR TITLE
The apology letter

### PR DESCRIPTION
**Pull request by mvgijssel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@
 - It's illegal to do a Direct Response when the opponent just served.
 - Players can do a Direct Response as you as you don't cross the net/[just](#vocabulary)
 
+### Apology letter
+Sportsmanship and being collegial are important values of the game. Therefore remorse should be shown, saying "Sorry" without laughing, in two situations where a player loses a life:
+- Ball goes into a weird angle due to hitting the side or corners of the table
+- Ball slightly hits the net drastically changing trajectory
+
+If no proper remorse is shown, can be determined with a vote, the player taking the life loses a life instead.
```
Comment (Willianvdv) [inline README.md]: `determined with a vote` → `determined by a vote`

**Review by Willianvdv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Willianvdv) [reply README.md]: The whole sentence is a bit weird

**Review by Willianvdv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RSO) [reply README.md]: "If no proper remorse is shown, a vote can determine whether the player-at-fault should lose a life instead."

Question: Will the other party be able to keep their live? Or do both players potentially lose their live if no remorse is shown and the party at fault is voted out as well?

Comment (mvgijssel) [reply README.md]: Yeah instead, otherwise two players can lose their lives at once creating the possiblity for a final with a single player.

**Review by Willianvdv — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mvgijssel) [reply README.md]: Also let's not vote for this everytime to keep the pace of the game high. Remember it's just an optimised version of tranditional 2-player pingping with some speed improvements.

Comment (Willianvdv) [reply README.md]: > pingping

😆

**Pull request merged**